codeship-services.yml

```diff
@@ -0,0 +1,69 @@
+testapp:
```

Comment:
Why is this `testapp` and not just `app`?

Reply:
Mainly because docker uses testapp and test.env for tests. I could switch it to just use main app, and have user need to populate local.env for local test runs.

Reply:
Maybe I'm missing something, but what I see is that `codeship-steps.yml` references service `test` for tests, but `testapp` for the Docker build/push.

Reply:
test service depends on testapp (among others).

Reply:
Oh, I see it now. In this file. Thanks.